tutorials/interactive/index.md

```diff
@@ -0,0 +1,814 @@
+---
+layout: tutorial
+title: "Adding Interactivity to your Flutter App"
+sidebar: home_sidebar
+permalink: /tutorials/interactive/
+---
+
+<div class="panel" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What you'll learn:</b>
+
+* How to respond to taps.
+* How to create a custom widget.
+* The difference between a stateless and stateful widget.
+
+</div>
+
+## Introduction
+
+How do you modify your app to make it react to user input?
+In this tutorial, you'll add interactivity to an app contains only
+non-interactive widgets.  Specifically, you'll modify an icon to make
+it tappable.  To accomplish this, you'll create a custom stateful widget
+that manages two stateless widgets.
+
+<a name="note"></a>
+<aside class="alert alert-info" markdown="1">
+<i class="fa fa-lightbulb-o"> </i> **Note:**
```

Comment:
The word "Note" doesn't really add anything here IMHO. I would remove prefixes that say "Note" throughout. That's just my opinion though, feel free to disregard. :-)